views/dashboard-application.hbs

```diff
@@ -267,7 +205,38 @@
                                 <option {{#if user.sizeLarge}} selected{{/if}}>L</option>
                                 <option {{#if user.sizeXLarge}} selected{{/if}}>XL</option>
                             </select> </div>
-
+                        <span class='row'></span>
+                        <div class='form-group col-md-3'>
+                            <label>Kit Preference #1*</label>
+                            <select class='form-control' name='kitRanking1'>
+                                <option {{#if user.pref1MotionAddOn}} selected{{/if}}>Motion Add-on</option>
+                                <option {{#if user.pref1AVAddOn}} selected{{/if}}>Audio + Visual Add-on</option>
+                                <option {{#if user.pref1TIBox}} selected{{/if}}>TI Box</option>
+                            </select> </div>
+                        <div class='form-group col-md-3'>
+                            <label>Kit Preference #2*</label>
+                            <select class='form-control' name='kitRanking2'>
+                                <option {{#if user.pref2MotionAddOn}} selected{{/if}}>Motion Add-on</option>
+                                <option {{#if user.pref2AVAddOn}} selected{{/if}}>Audio + Visual Add-on</option>
+                                <option {{#if user.pref2TIBox}} selected{{/if}}>TI Box</option>
+                            </select> </div>
+                        <div class='form-group col-md-3'>
+                            <label>Kit Preference #3*</label>
+                            <select class='form-control' name='kitRanking3'>
+                                <option {{#if user.pref3MotionAddOn}} selected{{/if}}>Motion Add-on</option>
+                                <option {{#if user.pref3AVAddOn}} selected{{/if}}>Audio + Visual Add-on</option>
+                                <option {{#if user.pref3TIBox}} selected{{/if}}>TI Box</option>
+                            </select> </div>
+                        <span class='row'></span>
+                        <div class='form-group col-md-3'>
+                            <a href="/boxes" target="_blank"><button class='file-upload'>
```

Comment:
Make sure to remove target

Reply:
Why should he remove the target? Wouldn't it make sense to have the boxes page open in a new tab so it doesn't interrupt their application?

Reply:
Yeah, I guess that makes sense, I didn't consider that - good point